org.lflang/src/org/lflang/generator/c/CCoreFilesUtils.java

```diff
@@ -0,0 +1,91 @@
+package org.lflang.generator.c;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.lflang.TargetProperty.SchedulerOption;
+
+public class CCoreFilesUtils {
```

Comment:
This should have a class comment with a short description of the class and author information.